standardtracer/bench_test.go

```diff
@@ -0,0 +1,65 @@
+package standardtracer
+
+import (
+	"fmt"
+	"sync/atomic"
+	"testing"
+)
+
+var tags []string
+
+func init() {
+	tags = make([]string, 1000)
+	for j := 0; j < len(tags); j++ {
+		tags[j] = fmt.Sprintf("%d", randomID())
+	}
+}
+
+type countingRecorder int32
+
+func (c *countingRecorder) RecordSpan(r RawSpan) {
+	atomic.AddInt32((*int32)(c), 1)
+}
+
+func benchmarkWithOps(b *testing.B, numEvent, numTag, numAttr int) {
```

Comment:
Thanks in advance for those, then.

I didn't want to do profiling and optimization until the API had stabilized... I guess we're there (or quite nearly there), so hopefully this isn't a waste of your time!